thrift.js

```diff
@@ -266,21 +376,13 @@ Thrift.prototype.resolveValue = function resolveValue(def) {
         return self.resolveListConst(def);
     } else if (def.type === 'ConstMap') {
         return self.resolveMapConst(def);
-    // istanbul ignore else
     } else if (def.type === 'Identifier') {
         if (def.name === 'true') {
             return true;
         } else if (def.name === 'false') {
             return false;
         }
-        // istanbul ignore if
-        if (!self.consts[def.name]) {
-            err = new Error('cannot resolve reference to ' + def.name + ' at ' + def.line + ':' + def.column);
-            err.line = def.line;
-            err.column = def.column;
-            throw err;
-        }
-        return self.consts[def.name].link(self).surface;
+        return self.resolveIdentifier(def, def.name).surface;
```

Comment:
you mean `resolveIdentifier`? If so, not possible. There are call sites to this function that don't use the same property on def.